README.md

```diff
@@ -30,24 +29,48 @@
 ### npm
 
 ```sh
-> npm install rif-communications
+> npm install @rsksmart/rif-communications
 ```
 
-### Use in Node.js
-
 ## Usage
 
+```js
+// 1. Import the library
+var RifCommunications = require('@rsksmart/rif-communications');
+
+// 2. Create a key or load from a secure storage to skip this step
+var key = await RifCommunications.createKey();
+
+// 3. Create peer id
+var peerId = await RifCommunications.createPeerIdFromJSON(key);
+
+// 4. Create peer information
+var peerInfo = await RifCommunications.createPeerInfo(peerId);
+
+// 5. Create your peer node
+var node = RifCommunications.createNode(peerInfo, host, port, sendMsgFunc);
+
+// 6. Connect to a bootnode using a multiaddress string
+await RifCommunications.connectToNode(node, BOOTNODE_ADDRESS);
```

Comment:
Maybe would be good to make clear that this is the initialisation part with sub heading, e.g:
## Usage
### Initialise
`...`
### Connect to another user
`...`